controller.go

```diff
@@ -106,7 +119,7 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type sandboxTable map[types.UUID]*sandbox
```

Comment:
Let's not re-introduce types.UUID at libnetwork level. Let's keep it a string. It is just a string. It doesn't exactly follow UUID specification anyways.